app/cronjob/trailingTrade/step/__tests__/get-indicators.test.js

```diff
@@ -2249,21 +2272,22 @@ describe('get-indicators.js', () => {
             indicators: {
               highestPrice: 10000,
               lowestPrice: 8893.03,
-              athPrice: 9000
+              athPrice: 9887.9
             },
             lastCandle: {
               symbol: 'BTCUSDT',
-              close: '15555.09000000'
+              close: '8900'
             },
             buy: {
-              currentPrice: 15555.09,
-              limitPrice: 15881.746889999999,
+              currentPrice: 8900,
+              limitPrice: 9086.9,
               highestPrice: 10000,
               lowestPrice: 8893.03,
-              athPrice: 9000,
-              athRestrictionPrice: 8100,
+              nextBestBuyAmount: 204983.33333333314,
```

Comment:
@rando128 
I have a question about the next best buy amount, but it seems your calculation is correct. 

So the parameters were as below:

- Last buy price: 9966.66666666667
- Last closed: 8900
- Buy trigger: 0.8929765886287623
- Total bought quantity: 1.5
- Total bought amount: 19900

- Next best buy amount =>  204983.33333333314

And I feel like this is a lot of money to buy for next. So I did a calculation.


Current price | 8900 |   |   |  
-- | -- | -- | -- | --
Grid Trade # | Amount | Quantity | Quote Amount | Last buy price
#1 | 10000 | 1 | 10000 |  
#2 | 9900 | 0.5 | 4950 |  
Sum |   | 1.5 | 14950 | 9966.66666666667
Next Best Buy amount | 8900 | 23.0318352059925 | 204983.333333333 |  
  |   | 24.5318352059925 | 219933.333333333 | 8965.2213740458

Is my calculation correct?

Reply:
@chrisleekr, what is your sellTrigger (i.e expected market rebound) for this calculation?